resources/vault_install_binary.rb

```diff
@@ -23,11 +23,13 @@
 default_action :install
 
 action :install do
-  directory new_resource.extract_to do
-    mode '0755'
-    owner new_resource.user
-    group new_resource.group
-    recursive true
```

Comment:
Doesn't seem like the cleanest way to do this, my syntaxy checker yelled at me for having >80 characters on that line.